src/python_testing/TC_ACL_2_11.py

```diff
@@ -160,12 +193,18 @@ async def test_TC_ACL_2_11(self):
             server_list = ret[endpoint][Clusters.Descriptor][Clusters.Descriptor.Attributes.ServerList]
             for server in server_list:
                 cluster = Clusters.ClusterObjects.ALL_CLUSTERS[server]
-                await dev_ctrl.ReadAttribute(dut_node_id, [endpoint,
-                                                           cluster.Attributes.GeneratedCommandList,
-                                                           cluster.Attributes.AcceptedCommandList,
-                                                           cluster.Attributes.AttributeList,
-                                                           cluster.Attributes.FeatureMap,
-                                                           cluster.Attributes.ClusterRevision])
+                data = await dev_ctrl.ReadAttribute(dut_node_id, [(endpoint, cluster.Attributes.GeneratedCommandList),
```

Comment:
i would like to refactor this list to be in common with the global_clusters list in a subsequent change.